docs/manual/customization.md

```diff
@@ -241,6 +240,86 @@ Using service IPs presents its own challenges:
 * We currently only support services with the ClusterIP type. These IPs may not be routable from outside the Kubernetes cluster.
 * The Service IP space is often more limited than the pod IP space, which could cause you to run out of service IPs.
 
+## Using DNS
+
+Using Pod IPs has the limitation that Pods might get a new IP address if they are created and sometimes using service IPs is not the right approach.
+FDB supports to use DNS in the cluster file since 7.1 and the operator can make use of that.
+*Note*: That this requires to build the operator image by yourself until we publish an image that contains the 7.1 library as primary library.
+Building the operator by yourself can be achieved with `docker build --build-arg FDB_VERSION=7.1.25 -t foundationdb/fdb-kubernetes-operator .` as an alternative you can add the following customization to inject the 7.1 library and use it as primary library:
```

Comment:
```suggestion
Using Pod IPs has the limitation that Pods might get a new IP address if they are recreated and sometimes using service IPs is not the right approach.
FDB supports to use DNS in the cluster file since 7.1 and the operator can make use of that.
*Note*: This requires the following customization to inject the 7.1 library and use it as primary library (see code example below). As an alternative you can build the operator image by yourself that contains the 7.1 library as the primary library.
Building the operator by yourself can be achieved with `docker build --build-arg FDB_VERSION=7.1.25 -t foundationdb/fdb-kubernetes-operator .`.
```